### ISSUE TITLE
Bug: Unable to import >=5.0.0-beta.11 in SvelteKit

### ISSUE THREAD

**Issue by enyo** [statelyai/xstate#4137]
### Description

I’m using the beta 10 fine in my project, but when I try to upgrade to beta 16 I get:

```
Error: Failed to resolve entry for package "xstate". The package may have incorrect main/module/exports specified in its package.json.
```

### Reproduction

https://codesandbox.io/p/sandbox/lucid-stitch-mj3x6t

### Additional context

I created a reproduction with the svelte kit skeleton. I don’t think it has anything to do with SvelteKit per se but with vite.

I know it’s beta, but because it worked with beta.10 I thought you should know about the regression.

**Comment by enyo** [comment]
If I add this to my vite config it works:


```js
// vite.config.js
const config = {
  // ...
  ssr: {
    noExternal: ['xstate'],
  },
  // ...
}
```

**Comment by Andarist** [comment]
The fix is live in `xstate@5.0.0-beta.18`. Thank you for the well-described issue 🙏

**Comment by enyo** [comment]
@Andarist thanks! Out of curiosity: what caused the problem?
